roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -6,9 +6,8 @@ script:
   indexed: on
 
 index:
-  number_of_shards: 1
-  number_of_replicas: 0
-  auto_expand_replicas: 0-2
+  number_of_shards: {{ es_number_of_shards | default ('1') }}
+  number_of_replicas: {{ es_number_of_replicas | default ('0') }}
```

Comment:
not exactly sure why, but because of the way we use this template, the values have to be quoted.  Please change this to be:
```
   number_of_shards: "{{ es_number_of_shards | default ('1') }}"
   number_of_replicas: "{{ es_number_of_replicas | default ('0') }}"
```
You'll have to submit a new PR since this one was merged.